app/src/main/java/org/p2p/wallet/bridge/send/ui/dialog/BridgeSendFeeBottomSheet.kt

```diff
@@ -84,15 +86,49 @@ class BridgeSendFeeBottomSheet : BaseDoneBottomSheet() {
         textViewFiatAmount.isVisible = false
     }
 
+    private fun ItemClaimDetailsPartBinding.bindWillGetAmountLine(title: String, fee: BridgeAmount) {
+        textViewTitle.text = title
+        textViewFiatAmount.text = fee.fiatAmount.orZero().asUsd()
+        bindFreeOrValue("${fee.tokenAmount.orZero().formatToken(fee.tokenDecimals)} ${fee.tokenSymbol}")
+    }
+
     private fun ItemClaimDetailsPartBinding.bindDetailsLineWithFee(title: String, fee: BridgeAmount) {
         textViewTitle.text = title
         textViewFiatAmount.text = fee.fiatAmount?.asUsd() ?: getString(R.string.bridge_info_transaction_free)
-        val formattedTokenAmount = fee.formattedTokenAmount
-        if (formattedTokenAmount == null) {
+        bindFreeOrValue(fee.formattedTokenAmount)
+    }
+
+    private fun ItemClaimDetailsPartBinding.bindDetailsTotalFee(title: String, fees: List<BridgeAmount>) {
+        val isFree = fees.isEmpty()
+        val totalFeesByToken = fees.groupBy { it.tokenSymbol }.values.map { bridgeFees ->
+            val feeForTokenData = bridgeFees.firstOrNull()
+            BridgeAmount(
+                tokenSymbol = feeForTokenData?.tokenSymbol.orEmpty(),
+                tokenDecimals = feeForTokenData?.tokenDecimals.orZero(),
+                tokenAmount = bridgeFees.sumOf { it.tokenAmount.orZero() },
+                fiatAmount = bridgeFees.sumOf { it.tokenAmount.orZero() }
```

Comment:
is it okay that you count tokenAmount here?

Reply:
no
will fix it thanks